content/english/anti-vegan/pests.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Pests"
+date: 2019-10-08T06:15:53+13:00
+lastmod: 2019-10-08T06:15:53+13:00
+draft: true
+description: "How to respond to the argument: Veganism fails to adequately address environmental concerns; specifically pest control."
+weight: 5
+---
+
+# Argument
+
+> Veganism is often criticised for failing to adequately address environmental concerns; specifically pest control.
+
+# Rebuttal
+
+## Quick Response
+
+> Environmental Ethics is complex and difficult. Even assuming we can reach agreement on the definition of pest it does not automatically follow that humans *ought* to control or eradicate this species.
+
+  
+## Detail - Analytic
+
+It is difficult to interpret how the central tenet of veganism *‘avoid exploitation of and cruelty to animals'* might be applied in complex environmental considerations. 
+
+However, before we try, numerous concepts require unpacking.
```

Comment:
`a few` rather than numerous? less intimidating is better 😄

Reply:
Yup.